includes/Modules/AdSense.php

```diff
@@ -838,4 +858,84 @@ public function check_service_entity_access() {
 		return true;
 	}
 
+	/**
+	 * Validates the report metrics.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $metrics The metrics to validate.
+	 *
+	 * @return null|WP_Error null if metrics are valid, otherwise WP_Error.
+	 */
+	protected function validate_report_metrics( $metrics ) {
+		if ( false === $this->is_using_shared_credentials ) {
+			return null;
+		}
+
+		$valid_metrics = apply_filters(
+			'googlesitekit_shareable_adsense_metrics',
+			array(
+				'ESTIMATED_EARNINGS',
```

Comment:
As per https://github.com/google/site-kit-wp/issues/5712#issuecomment-1222341102, please take the time to double-check the codebase and verify this list of metrics.